sig-governance.md

```diff
@@ -103,6 +103,7 @@ The process for closing a SIG/WG is as follows:
   - Move all appropriate github repositories to an appropriate archive or a repo outside of the Kubernetes org
     - Each subproject a SIG owns must transfer ownership to a new SIG, outside the project, or be retired
     - File an issue with kubernetes/org if multiple repos that need to be retired
+    - Existing SIG repos in kubernetes/community will be moved into kubernetes/community/archive
```

Comment:
Perhaps `Move the existing SIG directory to the [archive](/archive) in kubernetes/community`?

Reply:
Yes it'd be better if we were explicit here.

Reply:
Thanks will do!